app/build.gradle

```diff
@@ -523,6 +503,9 @@ android {
 
     // Needed to run espresso tests on release build.
     testBuildType project.ext.TEST_BUILD_TYPE
+    androidResources {
```

Comment:
is there a particular advantage of replacing `aaptOptions` with this. Also confirming that you have tested that these files are not making it to the apk with the new code.

Reply:
I don't exactly remember why I replaced `aaptOptions` with `androidResources`. It was perhaps deprecated but I can't find that information in the docs now. And I didn't test whether the files are present in the apk or not.

Reply:
added it back https://github.com/dimagi/commcare-android/pull/2526/commits/df53214f500c691c4f1ae1b8aace4a35e2c107e2 just for the sake of limiting changes. It seems to build fine with it.